h2o-py/h2o/estimators/estimator_base.py

```diff
@@ -257,6 +257,9 @@ def _resolve_model(self, model_id, model_json):
         m._metrics_class = metrics_class
         m._parms = self._parms
         m._estimator_type = self._estimator_type
+        m._start_time = model_json.get('output', {}).get('start_time', None)
```

Comment:
just to make are these always present?

Reply:
If not, the outcome is None. But yes, these are added to all models.

Reply:
Model has methods startClock and stopClock. As `Model` is parent to all other mdoels, these are always in place.